ButtonMerchant.xcodeproj/project.pbxproj

```diff
@@ -1161,7 +1166,7 @@
 					"$(inherited)",
 				);
 				GCC_SYMBOLS_PRIVATE_EXTERN = NO;
-				GCC_TREAT_WARNINGS_AS_ERRORS = YES;
+				GCC_TREAT_WARNINGS_AS_ERRORS = NO;
```

Comment:
Warnings are unnacceptable. Please revert.

Reply:
We are deprecating some methods and the amount variable and cannot silence these errors like you could in Objective-C.